.vscode/extensions.json

```
@@ -1,7 +1,9 @@
 {
   "recommendations": [
+    "Codecov.codecov",
     "davidanson.vscode-markdownlint",
     "esbenp.prettier-vscode",
-    "ms-playwright.playwright"
+    "ms-playwright.playwright",
+    "orta.vscode-jest"
   ]
```

I'm unsure whether this is supposed to be an exhaustive list, but here are some additional extensions which I'm using and seem relevant:
```suggestion
    "orta.vscode-jest",
    "dbaeumer.vscode-eslint",
    "GraphQL.vscode-graphql",
    "GraphQL.vscode-graphql-syntax",
    "hashicorp.terraform",
    "ms-vscode.vscode-typescript-next"
  ]
```

---

@nonparibus have disabled auto-merge just to give you a chance to see my suggestion before merging

---

@benwerner01 I already accepted the suggestion!